android/libraries/rib-base/src/main/java/com/badoo/ribs/core/Identifiable.kt

```diff
@@ -0,0 +1,6 @@
+package com.badoo.ribs.core
+
+interface Identifiable {
+
+    fun id(): String
```

Comment:
how about updating to val?

Reply:
Any strong reasons why?

Reply:
no strong reason just more nice API. You can keep it is as fun I am not insist.

Reply:
Done.

Reply:
**property** describes state.
**function** describes behaviour.

`id` is clearly not a behaviour, imho.